lib/connectors_sdk/fake_connector/http_call_wrapper.rb

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License;
+# you may not use this file except in compliance with the Elastic License.
+#
+
+# frozen_string_literal: true
+
+require 'connectors_sdk/base/http_call_wrapper'
+
+module ConnectorsSdk
+  module FakeConnector
+    class HttpCallWrapper < ConnectorsSdk::Base::HttpCallWrapper
+      SERVICE_TYPE = 'fake_connector'
+
+      def name
+        'Fake Connector'
+      end
+
+      def configurable_fields
+        [
+          {
+            'key' => 'third_party_url',
+            'label' => 'Third Party URL'
+          },
+          {
+            'key' => 'third_party_api_key',
+            'label' => 'Third Party API Key'
+          }
+        ]
+      end
+
+      def health_check(_params)
+        true
+      end
+
+      def document_batch(_params)
```

Comment:
I would still argue that we should do it in a "proper" way - with a client, if not extractor and adapter - but perhaps this "making the code into a template" should be scoped separately. I don't think it'll be a great effort though, as this FakeConnector needs no config and no adapter, just an extractor and custom_client.

Reply:
I think it would be nice to introduce at least an adapter for this connector. But yes, I think that should all be part of a subsequent PR.